docs/mux-logging-service.md

```diff
@@ -49,6 +49,20 @@ There are environment variables for the logging-mux DC config:
 * `FORWARD_LISTEN_HOST` - hostname to listen for secure_forward protocol log
   messages - this is the same as the FQDN in the mux server cert
 
+These are environment variables for the logging-fluentd daemonset config:
+
+* `USE_MUX_CLIENT` - `true`/`false` - default is `false` - if `true`, fluentd
```

Comment:
We don't yet know that mux works.

We know that for small deployments, adding a mux comes with additional management and monitoring complexity, more resources (cpu, memory, disk for its file buffer), etc.

If we can prove that fluentd to elasticsearch directly will simply not work, _and_ we can at the same time prove that mux is a good solution, _then_ we can get rid of the simple fluentd to elasticsearch case.